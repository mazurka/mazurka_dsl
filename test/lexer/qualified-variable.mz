Test <- Namespace/Variable

#{
  key <- NS/Var
}