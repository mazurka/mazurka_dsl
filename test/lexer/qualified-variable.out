{variable,1,'Test'}.
{assignment,1}.
{variable,1,{'Namespace','Variable'}}.
{map_begin,3}.
{atom,4,key}.
{assignment,4}.
{variable,4,{'NS','Var'}}.
{map_or_tuple_end,5}.